operators/endpointmetrics/manifests/prometheus/kubernetes-monitoring-alertingrules.yaml

```diff
@@ -1,962 +0,0 @@
----
-kind: ConfigMap
-apiVersion: v1
-metadata:
-  name: kubernetes-monitoring-alertingrules
-  namespace: open-cluster-management-addon-observability
-data:
-  kubernetes-monitoring-alertingrules.yaml: |
-    groups:
-    - name: general-rules
-      rules:
-      - alert: Watchdog
```

Comment:
I didn't see `Watchdog` alerting rule in new prometheus rule, do you forget that?
It is used to make sure the alerting mechanism is working, we have a test case(`alertforward/g0`) in e2e-kind for that, but it is disabled by default., see: https://github.com/stolostron/multicluster-observability-operator/blob/main/cicd-scripts/customize-mco.sh#L190
You can add `--focus alertforward/g0` to test that case.

Reply:
@morvencao 
I removed the alert rule for Watchdog since the triggered alerts don't make sense for a production env. I will double confirm this from Randy/Joydeep.
If this is removed, for this test case, the fix is to add a PrometheusRule to inject the Watchdog alert rule

Reply:
@morvencao 
I updated the e2e test case to handle the watchdog alert rule in *KS. Please help to review, thanks

Reply:
thanks @marcolan018 
the changes looks good to me.
Currently the kind-e2e testing doesn't cover the case, the reason is in kind env, we don't enable OCP route for alert forward target address. But the test case is useful QE to verify the alertforward function in local test environment.